unlock-app/src/__tests__/components/lock/ShowUnlessUserHasKeyToAnyLock.test.js

```diff
@@ -52,10 +53,10 @@ describe('ShowUnlessUserHasKeyToAnyLock', () => {
       },
     }
 
-    const wrapper = shallow(
+    const wrapper = rtl.render(
       <ShowUnlessUserHasKeyToAnyLock locks={locks} keys={keys}>
         Hide me
       </ShowUnlessUserHasKeyToAnyLock>)
-    expect(wrapper.text()).toEqual('')
+    expect(wrapper.queryByText('')).not.toBe(null)
```

Comment:
I am not sure this would be a good test. Would something like this be better:
```suggestion
    expect(wrapper.container.firstChild).toBeNull()
```